### DISCUSSION TITLE
Can Onedrive be Used instead of Gdrive

### DISCUSSION THREAD

**Discussion by icaruschan** [TheLastBen/fast-stable-diffusion#2251] [Q&A]
Is it possible to connect to Onedrive instead of gdrive

**Comment by TheLastBen** [comment]
colab works only with gdrive

**Comment by xsamueljr** [comment]
If you can somehow connect to OneDrive using Python like the `google.colab` module does with gdrive, yes you should be able to. The bad thing is that you would have to edit many parts of the notebook
